db/schema_tables.cc

```diff
@@ -2457,7 +2457,7 @@ std::vector<mutation> make_create_aggregate_mutations(schema_features features,
     m.set_clustered_cell(ckey, "return_type", aggregate->return_type()->as_cql3_type().to_string(), timestamp);
     m.set_clustered_cell(ckey, "state_func", aggregate->sfunc()->name().name, timestamp);
     m.set_clustered_cell(ckey, "state_type", state_type->as_cql3_type().to_string(), timestamp);
-    std::vector<mutation> muts = {m};
+    std::vector<mutation> muts = {std::move(m)};
```

Comment:
There's a chance this invokes the vector(initializer_list) constructor. You can't move from an initializer_list.

Reply:
It does.  godbolt.org tells me that indeed `m` is moved to the initializer_list and then it is copied into the vector.
I'll change that to muts.emplace_back(std::move(m)) to save the copy.

Reply:
fixed that (and more) in v2